src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
I'm happy to refactor this (add remove the SDK repo) if the rest of team agrees that there is value in sharing those objects and no need for and SDK even without a proper plugin dependency mechanism. Maybe I'm just missing a point. :)